### ISSUE TITLE
Terminal accessibility buffer does not read typed input

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15640]
Terminal live output is not announced.
As far as I tested on Windows with JAWS and NVDA, terminal aria-live event is not auto-announced. I typed "echo hello" in terminal, but there was no read-aloud feedback.

https://github-com/microsoft/vscode/-/171914-issue-1551817935
